### PR TITLE
Devel splitstack

### PR DESCRIPTION
**Pull request by JuhaHuiskonen**

### CODE REVIEW

File: lib/localrec/localized_reconstruction.py [hunk 1]
```diff
@@ -124,14 +125,13 @@ def angles_to_degrees(particle):
 def create_subparticles(particle, symmetry_matrices, subparticle_vector_list,
                         part_image_size, relax_symmetry, randomize, output,
                         unique, subparticles_total, align_subparticles,
-                        subtract_masked_map, do_create_star, filters):
```
Comment (delarosatrevin) [inline lib/localrec/localized_reconstruction.py:127]: Even if you have removed the option from the command line, I would ask you to keep it here in the function. This flag will allow us to not generate unnecessary star files when using localrec from Scipion.

File: examples/test_metadata.py [hunk 2]
```diff
@@ -23,93 +23,13 @@
...
-
-    for mic in ctfMd:
-        micDict[mic.rlnMicrographName] = mic
-
-    for particle in inputMd:
-        mic = micDict[particle.rlnMicrographName]
-        particle.copyValues(mic, *ctfLabels)
-
-    inputMd.write('particles_ctf.star')
+def main()
```
Comment (delarosatrevin) [inline examples/test_metadata.py]: Is the main function empty here?

Comment (JuhaHuiskonen) [inline examples/test_metadata.py]: This was not intentional... Need to recover the original file.

File: lib/localrec/localized_reconstruction.py (hunk 1 above)
Comment (JuhaHuiskonen) [inline lib/localrec/localized_reconstruction.py:127]: OK I will keep the option in the function.

**Pull request merged**